google/cloud/aiplatform/matching_engine/match_service_pb2_grpc.py

```diff
@@ -0,0 +1,134 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
@ReneeZhuGG These are manually compiled from protos and checked in. That means that if you change any of the service protos related to matching, we have to manually update these.

Can you let @dizcology, @morgandu and I know if that happens in the future?

Reply:
Sure, I will definitely do that.